src/lib/snarky_js_bindings/lib/snarky_js_bindings_lib.ml

```diff
@@ -1473,7 +1473,7 @@ module Circuit = struct
       Impl.constraint_system ~exposing:spec
         ~return_typ:Snark_params.Tick.Typ.unit (fun x -> main x)
     in
-    let kp = Impl.Keypair.generate cs in
+    let kp = Impl.Keypair.generate ~prev_challenges:0 cs in
```

Comment:
not true if we're creating a recursive circuits in snarkyjs no?

Reply:
to generate recursive circuits, snarkyjs goes through Pickles. this code is for plain circuits where we don't have anything making use of that `prev_challenges`